helpers/settings_test.go

```diff
@@ -19,147 +19,147 @@ var initSettingsTests = []initSettingsTest{
 	{
 		testName: "Basic Valid Production CF Settings",
 		envVars: map[string]string{
```

Comment:
change the tests to use the CSRFKey and SessionAuthentication

Reply:
Fixed, and added one test for use of `LegacySessionKey`.